pandokia/common.py

```diff
@@ -48,6 +48,14 @@ def check_auth():
         # If the config contains user_list, only those people are authorized.
         auth_ok = 1
 
+    # !!!
+    if not auth_ok:
+        f = open('/home/svc_etc/cds.pdk','a')
+        f.write(f'user is: "{user}", and cfg usr list is: "{cfg.user_list}"\n')
+        f.close()
+    return 1
+    # !!!
```

Comment:
Is this temporary?

Reply:
If not, what is cds.pdk?

Reply:
Yes, very temporary.  The `return 1` means it is doing no auth checking, so I def. can't commit like this.  The cds.pdk is a temp file to help debug whats coming in from the web side